build.py

```diff
@@ -38,6 +38,7 @@
 use_plugin("source_distribution")
 
 use_plugin("python.unittest")
+use_plugin("python.pytest")
```

Comment:
Why not? We test our project by two tools - perfect doublecheck.
On the other hand: if we want to get coverage with pytest we need to call it through `import pytest`. So we need installed pytest when we call tests for this plugin. As I understand we can do it by two ways:
1) use `build_depends_on('pytest')`
2) `use_plugin` + `plugin_depends_on('pytest')`